assets/templates/homepage/main-figures.tmpl

```diff
@@ -14,6 +14,7 @@
             <a href="https://www.ons.gov.uk/timeseriestool" class="tile__link">From our time series explorer</a>
         </span>
     </header>
+    {{ if .Data.HasMainFigures }}
```

Comment:
Noticed when I commented out multiple but not all URIs that there the tiles don't line up as they normally do (note the gap in the image). Again not introduced in this PR but whenever the last work on this was done it was possibly just missed. This is also not a mojor issue, low chance of people seeing this and next to no impact. So this is not blocking approval.

![Screenshot 2020-10-21 at 14 01 31](https://user-images.githubusercontent.com/47502916/96723112-143d6980-13a6-11eb-8576-e607acffed47.png)

Reply:
Looks like when we have the error message the height isn't growing. I suspect the employment/unemployment tile is even shorter when both fail to load. Probably fair to say it's not a major issue, but I will look into this a bit

Reply:
In my most recent commit ("set fixed height...") I've set a fixed height on medium viewport only - `height-md--42` for both the employment and unemployment content containers seems to help with ensuring the layout is more aligned in that viewport. No impact to any other viewport size 

![image](https://user-images.githubusercontent.com/23668262/96840306-2a990300-1442-11eb-8ead-e57be7be9d1d.png)

Reply:
(where only one emp/unemp figure loads)
![image](https://user-images.githubusercontent.com/23668262/96840971-11dd1d00-1443-11eb-874c-4dba019da67c.png)